tools/cassandra/handler.go

```diff
@@ -82,6 +82,9 @@ func handleUpdateSchema(config *UpdateSchemaConfig) error {
 }
 
 func handleSetupSchema(config *SetupSchemaConfig) error {
+	if config.CassPort == 0 {
```

Comment:
move this inside validateSetupSchemaConfig() - also add the same check to validateUpdateSchemaConfig and validateCreateKeyspaceConfig for consistency